provisioning/ubuntu-provision.sh

```diff
@@ -403,6 +403,15 @@ function install_azurecli() {
   apt-get install --yes --no-install-recommends azure-cli="${AZURECLI_VERSION}*"
 }
 
+## Ensure that azcopy is installed
+function install_azcopy() {
```

Comment:
Don't forget to call the function 😅

Reply:
Fixup commit pushed (https://github.com/jenkins-infra/packer-images/pull/898/commits/3f832ab738b637989ab74b5648af61362dc56ed2)